src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -76,8 +79,7 @@ public ActionSvcRestClientImpl(
       backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
   @Override
   public ActionPlanDTO createActionPlan(
-      final String name, final String description, final HashMap<String, String> selectors)
-      throws RestClientException {
+      final String name, final String description, final HashMap<String, String> selectors) {
```

Comment:
You should use the interface for collections , not the implementation , Map instead of HashMap

Reply:
:+1: This is a problem with the DTO for this action endpoint https://github.com/ONSdigital/rm-actionsvc-api/blob/master/src/main/java/uk/gov/ons/ctp/response/action/representation/ActionPlanPostRequestDTO.java#L29

We should update the DTO to use the interface then fix where we use it in the code. Will look at having a separate task for it